src/Bridge/VtkGlue/VTKImageToITKImage/CMakeLists.txt

```diff
@@ -5,12 +5,17 @@ project(VTKImageToITKImage)
 find_package(ITK REQUIRED)
 include(${ITK_USE_FILE})
 
+find_package(VTK REQUIRED)
```

Comment:
I think this will change stand-alone build behavior of this and other similar examples. They used to work fine with VTK8- and throw a configure warning with VTK9+. Now they will work fine with VTK9+, but fail to work with VTK8-.

Reply:
Assuming that `_vtk_prefix` will be undefined if being built stand-alone.

Reply:
Why do you think this first `find_package` will change that? VTK is required anyway in these cases.